BUILDING.md

```diff
@@ -280,6 +280,17 @@ When using the debug binary, core dumps will be generated in case of crashes.
 These core dumps are useful for debugging when provided with the 
 corresponding original debug binary and system information.
 
+Reading the core dump requires a gdb that was build on the same platform as the
```

Comment:
I'd suggest:
`Reading the core dump requires gdb built on the same platform core dump was captured on`